internal/ui/ephemeral_terminal_ui.go

```diff
@@ -126,14 +128,22 @@ func (h *ephemeralTerminalUI) flushLog() {
 	logWrapper, ok := log.Log.(*logger.LogrusLogger)
 	if ok {
 		fmt.Fprint(logWrapper.Output, h.logBuffer.String())
-		logWrapper.Logger.SetOutput(os.Stderr)
+		logWrapper.Logger.SetOutput(h.output)
 	} else {
-		fmt.Fprint(os.Stderr, h.logBuffer.String())
+		fmt.Fprint(h.output, h.logBuffer.String())
 	}
 }
 
 func (h *ephemeralTerminalUI) Teardown(force bool) error {
 	h.closeScreen(force)
-	ansi.CursorShow()
+	showCursor(h.output)
 	return nil
 }
+
+func hideCursor(output io.Writer) {
+	fmt.Fprint(output, "\x1b[?25l")
```

Comment:
can we get a method comment to add some context here? I have some general questions that might be implicit in the method name, but want to make sure...
* what does the `\x1b[?25l` code correspond to? 
* why does printing it to the specified output stream hide the cursor? 

These questions also apply to the showCursor method

Reply:
sorry @dakaneye , just as I merged your message popped up (I didn't refresh). I'll come back after the next couple of meetings and provide further context here 👍